src/components/map/VehiclePopupContent.svelte

```diff
@@ -0,0 +1,47 @@
+<script>
+	export let nextDestination;
+	export let vehicleId;
+	export let lastUpdateTime;
+	export let nextStopName;
+	export let predicted;
+
+	function formatLastUpdated(timestamp) {
+		const date = new Date(timestamp);
+		const now = new Date();
+		const secondsAgo = Math.floor((now - date) / 1000);
+
+		const minutes = Math.floor(secondsAgo / 60);
+		const seconds = secondsAgo % 60;
+
+		if (minutes > 0) {
+			return `${minutes} min${minutes > 1 ? 's' : ''} ${seconds} sec${seconds > 1 ? 's' : ''} ago`;
+		}
+		return `${seconds} sec${seconds > 1 ? 's' : ''} ago`;
+	}
+</script>
+
+// TODO: Add vehicle icon, add the nextStops times
```

Comment:
`<!-- TODO: Add vehicle icon, add the nextStops times -->` ;)

<img width="353" alt="image" src="https://github.com/user-attachments/assets/67d5db5f-c685-4bf6-af71-96f0dcee8717">